spec/garner/cache/identity_spec.rb

```diff
@@ -7,6 +7,13 @@
     subject.options_hash[:foo].should eq 'bar'
   end
 
+  it 'includes Garner.config.global_cache_options[:expires_in]' do
+    Garner.configure do |config|
+      config.global_cache_options = { expires_in: 5.minutes }
+    end
```

Comment:
This should be in a `before` block and you should clear this with a `Garner.reset!` or something similar in an `after` block.